tests/test_misc.py

```diff
@@ -476,6 +476,10 @@ def is_p2wpkh(output):
     assert only_one(fundingtx['vin'])['txid'] == res['wallettxid']
 
 
+@unittest.skipIf(
+    os.environ.get('TEST_DB_PROVIDER') == 'postgres',
+    "Postgres does not maintain insertion order on queries on its own"
+)
```

Comment:
That's strange?  Where does this fail?  It's one thing if the test relies on ordering, but another if the actual code does.

check_coin_moves seems to read from a json file written from the notifier, so I can't see how that gets wrong order...

Reply:
You're right, it's a commit I've had for a while and all of the sudden I can't reproduce the issue anymore. Will drop that commit from the PR.